.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
husky is a JS solution. We reused the existing hook from the fronted in Nu. Here, we can have sth like that:  `.hooks/pre-commit`

Reply:
didnt know that, thanks. Done :+1: